app/pages/OrganizationEditPage.tsx

```diff
@@ -1658,7 +1661,7 @@ class OrganizationEditPage extends React.Component<Props, State> {
         });
       } else {
         dataService
-          .APIDelete(path, { change_request: { status: "2" } } as any)
```

Comment:
Quick note on this: The second argument was incorrect, since it's supposed to be for extra HTTP headers to add to the API request, not for extra parameters to send. I specifically tested deactivating both a Resource and a Service after making this change, and it still works correctly.